pkg/rehearse/jobs.go

```diff
@@ -353,28 +438,32 @@ func replaceClusterProfiles(volumes []v1.Volume, profiles []config.ConfigMapSour
 type Executor struct {
 	Metrics *ExecutionMetrics
 
-	dryRun     bool
-	rehearsals []*prowconfig.Presubmit
-	prNumber   int
-	prRepo     string
-	refs       *pjapi.Refs
-	loggers    Loggers
-	pjclient   pj.ProwJobInterface
+	dryRun         bool
+	allowPeriodics bool
```

Comment:
In what case is this necessary?

Reply:
[Nevermind](https://github.com/openshift/ci-operator-prowgen/pull/175#pullrequestreview-249356080).

I would rather roll this out and have the results (and not have an extra flag), since the rehearse job is optional, but I'm fine with it.

Reply:
Let's keep it for now.